code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -487,8 +487,17 @@
 	glass_name = "glass of Pwr Game"
 	glass_desc = "Goes well with a Vlad's salad."
 
+/datum/reagent/consumable/pwr_game/on_mob_metabolize(mob/living/carbon/C)
```

Comment:
how could you oppress skeleton gamers like this im literally shaking and crying right now

Also, this message will trigger if you strap yourself to a stasis bed and then unstrap yourself from said stasis bed (and you qualify for the GAMER_GOD trait and don't already have the GAMER_GOD trait, of course), giving you a message that includes the phrase "As you imbibe", even though you added that Pwr Game to your system a long time ago.

Reply:
I hate reagent code!

Reply:
Why not do this on transfer?

Reply:
yeah, there should be no issue if you put it on reaction_mob (only allowing ingest, of course)